src/modules/sdlog2/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@
 #
 ############################################################################
 if (${OS} STREQUAL "nuttx")
-	list(APPEND MODULE_CFLAGS -Wframe-larger-than=1500)
+	list(APPEND MODULE_CFLAGS -Wframe-larger-than=2500)
```

Comment:
This is not good. It should only be increased as needed and the app stack adjusted as well. This warning is there for a reason, and we're low on RAM!